docs/zap-report.html

```diff
@@ -497,7 +497,7 @@ <h3>Alert Detail</h3>
   
   
 <tr bgcolor="#e8e8e8">
-<td width="20%" class="indent1">URL</td><td width="80%">https://s3.us-east-1.amazonaws.com/ustc-case-mgmt.flexion.us-documents-dev-us-east-1/%257BdocumentId%257D?AWSAccessKeyId=ASIAXQCLQG6G7CVIVUUH&amp;Expires=1557241534&amp;Signature=dNB6CgqR9S4z22E3nyTYYTVicG4%3D&amp;x-amz-security-token=AgoJb3JpZ2luX2VjEDcaCXVzLWVhc3QtMSJIMEYCIQDj2ncDEKdMrn%2Bq%2FRAX4mhiv9WfkqgBouiwG%2FKO%2FI7LWAIhAIsWq1USdFCQL%2FcX18Jk%2F%2F%2FPmHC%2Fyx8cSutYaabjMeMcKp0CCDAQABoMNTE1NTU0NDI0NzE3IgxovIYA6XuEHFHDdPkq%2BgECNf%2FWgeJUNkpEPcZ9VIeJi9T48i1fTHHdNVhzg9GqaKvN0DkYTO6pYyolAUuGdlj%2BZx9XLlNCQFS1dca2NI%2FKQHPq7R6DwEguhstjQiPnXgwcsU23%2Fb0OmcstUfYw5GEj5Qah5dImJaPFVcOXBcJNgfhdZhyt3qI0FnpQkIIEKO5KPl33yMZcnOuNmkHE%2BTFcYltAEgVoTucSi4nv7Z2FUiDczp8jecd5yp0b9ezhXJkWv3tV%2BRfcYWRh%2F5CWblAAVvVnCGlFLUBpy027ZtCiG81eNCAZeN8BFy1DWs2MKSIFBN2%2F74HDhd2V%2Fnb%2B1wODOZFvxdmCQtD1ML%2B4xuYFOrMBqnw2RDrKfWnrJj8W5IX2JqjpQiIDtt6rtXpp9suKZmxCJ2XKe9u%2F1%2FlfwgXwJM7%2BOT83QySN0iQUuerKmWSv2UXpqXP8EAr6fN9DNS0Wrml04%2BYK2HnK5IOC3UUCf3MDICBLUuCSazpFLr58CVbwQJ7F08iKzqYRZvgLcYYWngV0lyDlxP1C%2BMKUAc6lZU1Wh59smIGezDduEtALka1RCsU6jW0ST3FZb9i1UTa%2FVEBW%2BWg%3D</td>
+<td width="20%" class="indent1">URL</td><td width="80%">https://s3.us-east-1.amazonaws.com/ustc-case-mgmt.flexion.us-documents-dev-us-east-1/%257BdocumentId%257D?AWSAccessKeyId=ASIAXQCLQG6GS2SOBDFH&amp;Expires=1557842330&amp;Signature=%2Bci5dSUayzPjN2%2F0xpOh3yUvLiE%3D&amp;x-amz-security-token=AgoJb3JpZ2luX2VjEN7%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJGMEQCIB6A6B%2BMmAWOC4UFhNw26SZHPdybmpsSgWNMA0L9BXHsAiAioLnwN%2B4iSYtF0tfvQpLMK8p6GcmDP7uQtLkpg4Wa3CqmAgjX%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F8BEAAaDDUxNTU1NDQyNDcxNyIM5bSvjsCl21iPgMfoKvoBQxQSAsUJe03F6LxBHm9QKgTOzcgeZNeYY8UnnvF80kpV2j0yrQFXl1uxXSK0CSojPujacBOXyOPNSfohQbBB%2Bf2ykSjP3PyYGmt6dSZo%2BJ1e1H7ogV7s7n7ZSkMQJztDAwN1%2F1XQ7F5tDldfEXZhImpLHj8D%2Bd5PrKj8kbphlUcpSGxm7NWbyyxg04IUhchKdNr9dKBcYFSxEs1P6TX6u5xZp2m%2BJtwLgS6IubaxMKo1jFkqXJQCS3MzZzgms%2Fa1eN5VaZXNzDJZ1Kr7npukkqOB%2F6h4jDVFpdNFcGRFTxuywi%2B%2BLWXkdm9kzo%2FeFF7ixRxT4reF%2FdbjcjDVkevmBTq1AWqCHamTRGbbvx1lubZLQaPEizxmrCOkOtupV3sv2ElpqWYuvWYa4AjH%2FwX1n04KZo8fVqQo6Kc6aSVYYCGyQHBTypBu2V1nwRBcvk%2FjyX1OXXM0xpWzuzDwMdKYBMb9M8bj5PwUDmUbX40KvVvg2juqCRlTpKEKZgho%2F0mu0uRPZAQdDMp8mpaUQQoXqzsu24GQGIYcOgo%2Bc%2FpAtStb0YAjOyuPB8IjP3%2BVQ5c3piOAq2G4LJQ%3D</td>
```

Comment:
The build script concludes by providing `http://localhost:1234/log-in` as the URL to use for the site. This should presumably read `http://localhost:1234/` now, because using the provided URL redirects to `http://localhost:1234/mock-login?redirect_uri=http%3A//localhost%3A1234/log-in`.

Reply:
Updated!